pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -30,6 +30,21 @@ import (
 // Code is the Status code/type which is returned from plugins.
 type Code int
 
+// NodeScore represents the score of a particular node. The higher the score, the higher
+// the node is ranked during scheduling.
+type NodeScore struct {
+	// Name of the host
+	Host string
+	// Score associated with the host
+	Score int
+}
+
+// NodeScoreList declares a list of nodes and their scores.
+type NodeScoreList []NodeScore
```

Comment:
Based on the above implementation, we expect all NodeScores at a particular index to belong to the same node. In that case, it seems unnecessary to store node name for each `NodeScore`. An alternative approach is to use a map from a node name to a list of its scores. Each element of the list of scores contains score returned by one of the score plugins.

Reply:
I think it is better to keep the map from plugin name to list of scores, that way it is easier to implement the NormalizeScore extension point which expects a list of scores of all the nodes.